frontend/src/v5/store/viewer/viewer.sagas.ts

```diff
@@ -42,19 +42,36 @@ function* fetchData({ teamspace, containerOrFederation, project }: FetchDataActi
 
 		const isFederation = federationsWithoutStats.some(({ _id }) => _id === containerOrFederation);
 
+		const fetchContainerStatsHasError = { value: false };
+
 		if (isFederation) {
 			yield put(FederationsActions.fetchFederationStats(teamspace, project, containerOrFederation));
 			yield take(FederationsTypes.FETCH_FEDERATION_STATS_SUCCESS);
 			const federation: IFederation = yield select(selectFederationById, containerOrFederation);
 
 			for (const containerId of federation.containers) {
-				yield put(ContainersActions.fetchContainerStats(teamspace, project, containerId));
+				yield put(ContainersActions.fetchContainerStats(
+					teamspace,
+					project,
+					containerId,
+					null,
+					() => { fetchContainerStatsHasError.value = true; },
```

Comment:
this whole thing is kind of weird, the saga is waiting for a FETCH_CONTAINER_STATS_SUCCESS to be dispatched. In the case of a an error is that event dispaatched?